### DISCUSSION TITLE
[Announment] Welcome new PMC Member: Le Zhang

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#4143] [Announcements]
Hi Community,

On behalf of the Apollo PMC, I am pleased to announce that Le Zhang a.k.a lepdou, has been voted in as a new Apollo PMC Member.

Please join me to say congratulations to him!

@lepdou, would you please briefly introduce yourself to the community?

Best Regards,
Jason

**Comment by zouyx** [comment]
Congratulation to @lepdou and welcome!
